doc/pages/source/documentation/index.md

```diff
@@ -64,9 +66,20 @@ Dynflow has been developed to be able to support orchestration of services in th
 -   **World** - the universe where the Dynflow runs the code: it holds all
     needed configuration. Usually there's only one world per Dynflow process,
     besides configuration it also holds `Persistence`, `Logger`, `Executor` and
-    all the other objects necessary for action executing.
+    all the other objects necessary for action executing. This concept
+    allows us to avoid globally shared state. Also, the worlds can
+    talk to each other, which is helpful for production and
+    high-availability setups, having multiple worlds on different
+    hosts handling the execution of the execution plans.
```

Comment:
imo it should be "having multiple world on different hosts handle the execution..."